pipelines/pipeline.yml

```
@@ -0,0 +1,20 @@
+parameters:
+  - name: platform
+    type: string
+  - name: pool
+    type: 'object'
+    default: {}
+
+jobs: 
+  - job: ${{ format('Build_{0}', parameters.platform) }}
+    displayName: ${{ format('Build {0}', parameters.platform) }}
```

I think here also using the `format` function isn't really needed:
```suggestion
  - job: Build_${{parameters.platform}}
    displayName: Build ${{parameters.platform}}
```